lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs

```diff
@@ -147,6 +149,10 @@ import GHC.Generics
 import GHC.TypeLits
     ( KnownNat, Nat, Symbol, natVal )
 
+import Data.Type.Equality
+import Type.Reflection
+    ( typeOf )
```

Comment:
Why are these imports are separated from the rest? 
Also, `Data.Type.Equality` is importing stuff in context implicitly, let's make the import fully explicit for readability.

Reply:
Done.